docs/content/direct/common-setup-hub-chart.md

```diff
@@ -0,0 +1,80 @@
+# Common Setup - hub chart
+
+For more details please see [here](./hub-chart.md).
+
+This documents explains how to use KubeStellar Hub chart to deploy a new instance of KubeStellar
+with a choice of user-defined KubeFlex Control Planes (CPs).
+
+The information provided is specific for the following release:
+
+```shell
+export KUBESTELLAR_VERSION=0.23.0-alpha.2
+```
+
+## Create a Kind cluster
+
+The setup of KubStellar via the Hub chart requires the existance of at least one cluster
+to be used for the deployment of the chart.
+
+For convenience, a new local Kind cluster that satisfies the requirements for KubeStellar setup
+and that can be used to exercises the [examples](./examples.md) can be created with the following command:
+
+```shell
+sudo su <<EOF
+bash <(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/v${KUBESTELLAR_VERSION}/scripts/create-kind-cluster-with-SSL-passthrough.sh) --name kubeflex --port 9443
+EOF
+```
+
+## Install KubeStellar Hub chart
+
+A KubeStellar Hub installation compatible with the common setup suitable for Common Setup described in the [examples](https://docs.kubestellar.io/release-0.22.0/direct/examples/) could be achieved with the following command:
+
+```shell
+helm upgrade --install ks-hub oci://ghcr.io/kubestellar/kubestellar/hub-chart --version $KUBESTELLAR_VERSION \
+  --set-json='ITSes=[{"name":"its1"}]' \
+  --set-json='WDSes=[{"name":"wds1"}]'
+```
+
+Remember to include the option `--set "kubeflex-operator.isOpenShift=true"`, when deploying in an OpenShift cluster.
+
+## Obtain the kubeconfigs to access the KubeFlex Control Planes
+
+After the KubeStellar Hub has fully deployed,

```

Comment:
This glosses over the waiting that is spelled out in the other copy of this procedure.

Reply:
The reason is that waiting for the CP is, I believe, necessary and sufficient for getting the kubeconfigs but not sufficient for the cluster registration/joining operation. There is not a simple way to determine when OCM is ready not kubestellar.

For the moment I purposely was vague.